### PR TITLE
feat: New spec "/sys/class/tty/console/active" and parser

### PR DESCRIPTION
**Pull request by qinpingli**
### All Pull Requests:

Check all that apply:

* [X] Have you followed the guidelines in our Contributing document, including the instructions about commit messages?
* [ ] Is this PR to correct an issue?
* [ ] Is this PR an enhancement?

### Complete Description of Additions/Changes:

*Add your description here*

### CODE REVIEW

**Comment by qinpingli** [comment]
@xiangce @JoySnow please help review, thanks!

**Review by JoySnow — COMMENTED** [review]
Hi @qinpingli , the parser LGTM, just this spec `/sys/class/...` seems not fit in well with this `/sys/module` related parser file.  Also, there is another parser file _insights/parsers/sys_kernel.py_ for `sys/kernel` specs. 
Based on there practices, what about opening a new parser file for these `sys/class` specs , say _insights/parsers/sys_class.py_ ?

**Comment by qinpingli** [comment]
> Hi @qinpingli , the parser LGTM, just this spec `/sys/class/...` seems not fit in well with this `/sys/module` related parser file. Also, there is another parser file _insights/parsers/sys_kernel.py_ for `sys/kernel` specs. Based on there practices, what about opening a new parser file for these `sys/class` specs , say _insights/parsers/sys_class.py_ ?



> Hi @qinpingli , the parser LGTM, just this spec `/sys/class/...` seems not fit in well with this `/sys/module` related parser file. Also, there is another parser file _insights/parsers/sys_kernel.py_ for `sys/kernel` specs. Based on there practices, what about opening a new parser file for these `sys/class` specs , say _insights/parsers/sys_class.py_ ?

Good suggestion, I'll update it. Thanks!

**Comment by qinpingli** [comment]
@JoySnow Updated, please help take a look again, thanks!

**Review by JoySnow — APPROVED** [review]

File: insights/parsers/sys_class.py [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+        >>> tty_console_active.devices
+        ['tty0', 'ttyS0']
+    """
+
+    def parse_content(self, content):
+        if len(content) > 1:
+            raise SkipComponent("This should be an one line file")
+        if not content:
+            self.devices = []
```
Comment (xiangce) [inline insights/parsers/sys_class.py:41]: Please add comment and docstring for this empty case, see https://github.com/RedHatInsights/insights-core/pull/4166/files#r1680633434

**Pull request merged**